plugins/mac-capture/window-utils.h

```diff
@@ -6,6 +6,7 @@
 
 struct cocoa_window {
 	CGWindowID window_id;
+	NSNumber *owner_pid;
```

Comment:
I would suggest storing `intValue` or whatever "integer"-like representation that value has, otherwise you're missing a whole bunch of `[_ retain]` + `[_ release]` calls below

Reply:
Done